Makefile

```diff
@@ -19,15 +19,15 @@ fmt-check:
 
 .PHONY: check
 check:
-	cargo check --examples --tests
+	cargo check --all-targets
 
 .PHONY: check-without-features
 check-without-features:
 	cargo check --manifest-path "scylla/Cargo.toml" --features ""
```

Comment:
I think this one should be updated, too.

Reply:
Done